### PR TITLE
Change pool_exclude with change in API.

### PR DESCRIPTION
**Pull request by vvenkates27**
Modifying pool_exclude based on recent çhanges to the API.

### CODE REVIEW

File: src/aiori-DAOS.c [hunk 1]
```diff
@@ -668,7 +674,12 @@ kill_daos_server(IOR_param_t *param)
 	targets.rl_nr.num	= 1;
 	targets.rl_nr.num_out	= 0;
 	targets.rl_ranks	= &rank;
-	rc = daos_pool_exclude(pool, &targets, NULL);
+
+        s = strdup(param->daosPoolSvc);
+        svc.rl_ranks = svc_ranks;
+        ParseService(param, sizeof(rank)/ sizeof(svc_ranks[0]), &svc);
```
Comment (liw) [inline src/aiori-DAOS.c]: Shouldn't it be "sizeof(svc_ranks) / sizeof(svc_ranks[0])"?

File: src/aiori-DAOS.c [hunk 2]
```diff
@@ -668,7 +674,12 @@ kill_daos_server(IOR_param_t *param)
 	targets.rl_nr.num	= 1;
 	targets.rl_nr.num_out	= 0;
 	targets.rl_ranks	= &rank;
-	rc = daos_pool_exclude(pool, &targets, NULL);
+
+        s = strdup(param->daosPoolSvc);
```
Comment (liw) [inline src/aiori-DAOS.c]: Why is this strdup() required, please?

Comment (vvenkates27) [inline src/aiori-DAOS.c]: Left-over from previous implementation..

File: src/aiori-DAOS.c (hunk 1 above)
Comment (vvenkates27) [inline src/aiori-DAOS.c]: Yep Fixed now.

**Comment by vvenkates27** [comment]
I have merged this with a different pull request. Please check that one. Thanks

**Pull request closed without merging**